linchpin/ansible_runner.py

```diff
@@ -156,12 +181,19 @@ def ansible_runner(playbook_path,
                       ssh_extra_args=None,
                       sftp_extra_args=None,
                       scp_extra_args=None,
+                      start_at_task=None,
                       verbosity=verbosity,
                       check=False,
                       diff=False
                       )
 
-    if ansible24:
+    if ansible_version >= 2.8:
```

Comment:
@14rcole : Are you sure its >=2.8? I remember you saying there was a problem with 2.7.10 version?

Reply:
The problem with the 2.7.10 version and the problem with 2.8 are the same.  However, I had trouble reproducing the issue with 2.7.10 and I could consistently reproduce it with 2.8